eBook/examples/chapter_12/read_files.go

```diff
@@ -20,6 +20,7 @@ func main() {
 			fmt.Printf("The file %v does not exist!\n", flag.Arg(i))
 			break
 		}
+		defer fin.Close()
```

Comment:
`defer` in a `for` loop is not ideal, should just have it closed as the last statement in the loop (i.e. after line 27).

Reply:
Great point. Updated now